src/mainboard/amd/romecrb/fixed-dtfs.dts

```diff
@@ -13,43 +13,43 @@
             area@0 {
                 description = "Boot Blob";
                 compatible = "ore-bootblob";
-                offset = <0x3f0000>;
-                size = <0x10000>;
+                offset = <0x3e0000>;
```

Comment:
was this changed for your board? I will need to test these numbers on CRB I guess?

Reply:
This matches your fork